zh/deploy-tidb-operator.md

```diff
@@ -249,5 +249,5 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
     {{< copyable "shell-regular" >}}
 
     ```shell
-    helm upgrade tidb-operator ./tidb-operator
+    helm upgrade tidb-operator ./tidb-operator --namespace=tidb-admin
```

Comment:
no `Customize TiDB Operator` in ZH doc, 😂

Reply:
The Chinese version of the **Customize TiDB Operator** section will be added by https://github.com/pingcap/docs-tidb-operator/pull/1266.